### PR TITLE
Botany adjustments/re-addition/fixes/whatever (RE-UPLOAD)

### LINKED ISSUE #6407: Deep-frying an ear of corn won't make corn chips
**Problem Description**:
Putting an ear of corn into the deep fryer produces deep-fryed ear of corn instead of corn chips like the code and wiki say it is supposed to.

**What did you expect to happen**: 
Corn put in the deep fryer should become corn chips.

**What happened instead**:
Deep fryed corn was produced instead of corn chips.

**Why is this bad/What are the consequences:**
The code says that the corn should become chips, though the botany update might have changed the corn id or something along those lines.

**Steps to reproduce the problem**:
Get an ear of corn and put it into the deep fryer, then wait for it to come out.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: I found this in the code for the deep fryer:
86 /datum/deepfryer_special/corn_chips 
87 	input = "corn" 
88 	output = /obj/item/weapon/reagent_containers/food/snacks/cornchips

### LINKED ISSUE #6561: Cannot make rice in new botany
**Problem Description**:
Grinding rice stalks is not making rice, and thus cannot be used in recipes

**What did you expect to happen**: 
Rice is a needed reagent. Can't make it right now.

**What happened instead**:
Could grind rice in old botany.

**Why is this bad/What are the consequences:**
It's annoying, but not game breaking.

**Steps to reproduce the problem**:
Grind rice stalks. Get plant matter, not rice. Rice is missing as a reagent in the stalk (at least, thats what the mhelp reported when I prompted them to check)

### PR DESCRIPTION
**Pull request by FalseIncarnate**
Re-upload of #6411 because .dmi conflicts aren't worth the hassle and it allowed me to split one of the existing commits into two more atomic ones.

Fixes #6407 
- Corn and bananas will now properly be deepfried into corn chips and fried bananas respectively.
  - Variants of these plants (like mimanas) also will work as a substitute due to how subtyping works. This is fine / intentional.

Fixes #6561 
- Rice can be properly ground into it's reagent form again.

Re-adds Tray Lids
image
- Hydroponics trays once again have tray lids which can be toggled via a verb or alt-click on the tray. These lids only protect the plant from accidental harvesting and bees. Especially bees.
  - Lid lights are not re-added, because they no longer are necessary.

Re-slows Plant Aging
- Plant aging has been readjusted to better reflect the longer average shift of Paradise in comparison to TG station.
  - Instead of a plant's age increasing every cycle, it once again increases every 2rd cycle (meaning a plant takes 2x as many cycles to reach maturity than it does currently).
  - This makes botany faster than the old system, but won't overwhelm newer players too quickly by being too fast. You can always lower the maturation and production times to speed things up anyways.

:cl:
bugfix: Corn and bananas can be properly deepfried into their special results again.
bugfix: Rice can be properly ground into a reagent again.
rscadd: Hydroponics trays once again are equipped with Bee-proof lids to keep unwanted pollination to a minimum. This only works if you actually toggle the lid closed though!
tweak: Plants age slightly slower, so your space farming should no longer feel like it outpaces a meth'd up blue hedgehog in sneakers.
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
Not to influence adding more to your PR (It looks good)... But any chance of a control A + delete on growth serum? - https://nanotrasen.se/phpBB3/viewtopic.php?f=12&t=9900

**Comment by FalseIncarnate** [comment]
Out of scope, that's more a revert request that would warrant it's own PR for its own dedicated discussion. Adding that to this would just result in this getting delayed unnecessarily and possibly denied while it remains.

**Comment by SamHPurp** [comment]
Worth a try. 👍

**Comment by uraniummeltdown** [comment]
Please either remove the aging change or tone it down to 2x, old botany was mostly "spend most of your time staring at the screen waiting for the plant to grow".

**Comment by Fox-McCloud** [comment]
I'll re-iterate what a few others have said in the other PR.

>Staring at your screen waiting 3 times as long for plants to grow is not very fun, if you must make aging slower make it age every 2 cycles

I'm still not exactly keen on the speed adjustments for reasons stated above, especially with how the current pace gives something for people to do a lot more; it also gives player the very real option to focus on what the chef needs and wants for part of the shift and to engage in a lot of research during the latter part of the shift.

The botany lids thing is ok, though comes across as a bit pointless.

**Comment by jughu** [comment]
Yeah the tray lids are useless

File: code/modules/hydroponics/hydroponics.dm [hunk 1]
```diff
@@ -904,6 +928,9 @@
 /obj/machinery/hydroponics/attack_hand(mob/user)
 	if(issilicon(user)) //How does AI know what plant is?
 		return
+	if(lid_state)
+		to_chat(user, "<span class='warning'>You can't reach the plant through the cover.</span>")
```
Comment (Iamgoofball) [inline code/modules/hydroponics/hydroponics.dm]: literally no point for a lid

**Comment by Iamgoofball** [comment]
Lids are useless.

Slowing it down for the sake of slowing it down is retarded.

**Comment by fludd12** [comment]
I don't normally agree with goof, but I kinda prefer faster botany. Watching-plants-grow-simulator wasn't very fun. What you could do, though, is make plant production rate have a similar extraction restriction in unupgraded genetic machines, since production rate can get to 1 from roundstart.

**Comment by FalseIncarnate** [comment]
Sped up the rate a little, now it ages every other cycle (2 cycles per age up). One of the big intents here was to make the pacing of botany a bit more consistent with our round-lengths and to also make it less overwhelming for newer players as botany IS a role that is available as soon as you join Paradise for the first time (so a number of botanists are still relatively wet behind the ears regarding SS13). Veteran botanists will have a bit more time between harvests to mutate and engineer their perfect crops as well, and will likely just lower the maturation and production times to speed things up on their own terms.

**Comment by Kyep** [comment]
@FalseIncarnate what I'd like most from botany:
- make bluespace fruit harder to get (not really picky how, but its too common currently)
- when someone throws bluespace fruit at someone else, and teleports them, it should generate both a VV attack_logs log, and a ATTACK: log message for admins with attack log preferences enabled

**Comment by Fox-McCloud** [comment]
Since there's no clear consensus yet on the speed/cycle, with very mixed (and very few) opinions; why don't you split off the cycle delay to another PR; if this is just fix+tray lids, this could be merged rather quickly.

**Comment by FalseIncarnate** [comment]
It seems the consensus was to change the aging to be every second cycle, which I did. Far as I can tell, there is nothing holding this back other than goof not liking it at all.

**Comment by Aurorablade** [comment]
this seems okay.

**Comment by FalseIncarnate** [comment]
New commit fixes rice not grinding into the rice reagent.

**Comment by Jountax** [comment]
@FalseIncarnate It's impossible to alter maturation rates.

**Pull request merged**